bitcash/crypto.py

```diff
@@ -20,3 +23,110 @@ def ripemd160_sha256(bytestr):
 
 
 hash160 = ripemd160_sha256
+
+
+def sha512(bytestr):
```

Comment:
I think that only the minimum should contain public methods. This could be `_sha512()`.